src/app/core/service/platform/auth.service.ts

```diff
@@ -7,9 +7,24 @@ import {API_ENDPOINTS} from '../../utils/constants.utils';
   providedIn: 'root'
 })
 export class AuthService extends PlatformService {
-  forgotPasswordOtp(email) {
-    const requestBody = {email: email};
-    return this.post(API_ENDPOINTS.AUTH_OTP_FORGOT_PASSWORD, requestBody);
+  sendForgotPasswordOtp(email) {
+    const requestBody = {purpose: 'FORGOT_PASSWORD', email: email};
+    return this.post(API_ENDPOINTS.AUTH_OTP, requestBody);
+  }
+
+  verifyMobile(mobile) {
+    const requestBody = {purpose: 'VERIFY_MOBILE', mobile: mobile}
```

Comment:
Can purpose be made a constant?

Reply:
No, not needed now, as we are using it only in one place here